### ISSUE TITLE
How to reach mAP 0.69 on validation set

### ISSUE THREAD

**Issue by NoOneUST** [TAMU-VITA/FasterSeg#30]
After searching and training on the val set, I can only get 0.69 mAP on training set, on val set it is only 0.64. Do I need to try search and train many times in order to get the claimed result? Or it needs more than 600 epochs?

**Comment by chenwydj** [comment]
Hi @NoOneUST!

Thank you for your interest in our work!

You could visualize the searched architecture in tensorboard. If the searched architecture does not make sense (e.g. not choosing enough 3x3 convolution or pooled convolution), then you may need to tune the latency constraint or use early stopping during the search process.
